client/components/App.jsx

```diff
@@ -14,11 +13,7 @@ class GenDesc extends React.Component {
   }
   componentWillMount() {
     window.addEventListener('house_view', e => {
-      fetch(
-        `http://ec2-52-14-98-3.us-east-2.compute.amazonaws.com/houses/${Math.floor(
-          Math.random() * 98
-        )}`
-      )
+      fetch(`http://localhost:6001/houses/${Math.floor(Math.random() * 98)}`)
```

Comment:
You'll want to use environment variables to determine the domain name here, too, so that you don't have to change it back and forth between some `ec2` URL and `localhost` as you move from `prod` to `dev` to `test` environments a lot.

Reply:
I'm just doing the port for now and will update tomorrow once I get some of this stuff deployed to EC2 and have a better idea of what those urls will look like.